plugin/NERD_commenter.vim

```diff
@@ -986,19 +987,22 @@ function s:CommentLinesSexy(topline, bottomline)
         call cursor(a:topline, 1)
         execute 'normal! O'
         let theLine = repeat(' ', leftAlignIndx) . left
-
         " Make sure tabs are respected
         if !&expandtab
            let theLine = s:ConvertLeadingSpacesToTabs(theLine)
         endif
         call setline(a:topline, theLine)
-
+        " let result = confirm(theLine)
+        " execute "confirm"
```

Comment:
Where these lines left over from debugging perhaps? I don't see how they explain anything that would warrant a comment. If they don't serve a purpose could you remove them? Thanks.

Reply:
Thanks for fixing this @desperadoshi.